simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp

```diff
@@ -454,6 +478,14 @@ void SimplePlanningSimulator::on_timer()
   publish_turn_indicators_report();
   publish_hazard_lights_report();
   publish_tf(current_odometry_);
+
+  if (enable_pub_steer_) {
+    publish_steering(current_steer_);
+  }
+
+  if (vehicle_model_type_ == VehicleModelType::ACTUATION_CMD) {
```

Comment:
The `VehicleModelType` should not be used here for extensibility. Please define steering_publish_type in constructor, use it here.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8415/commits/ed78398b7705c83460c9089f2aef53019f552e9f

Reply:
added `virtual bool shouldPublishActuationStatus() const { return false; }`